src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -2230,7 +2230,7 @@ TEST_P(conv_int8_scale_activation_quantize_i8_eltwise_fp32, basic) {
         reorder("reorder_bfyx", "sum", p.default_format, data_types::f32)
     );
 
-    tolerance = 1.f;
+    tolerance = 2.f;
```

Comment:
Why this tolerance might relevant to this change?

Reply:
New test cases led to the different data generated for fusion tests and tolerance 1.0 was not enough anymore.